src/map/script.c

```diff
@@ -7437,16 +7437,15 @@ BUILDIN(grouprandomitem) {
 }
 
 /*==========================================
- *
- *------------------------------------------*/
-BUILDIN(makeitem)
-{
-	int nameid,amount;
-	int x,y,m;
+* makeitem <item id or name>,<amount>,"<map name>",<X>,<Y>;
+*------------------------------------------*/
+BUILDIN(makeitem) {
```

Comment:
according to Coding Style Chapter 3(https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-3-placing-braces-and-spaces)
It should be
```
BUILDIN(makeitem)
{
```